test/utilities/ajv.js

```diff
@@ -12,25 +12,13 @@ let schemasLoaded = false;
 // METHODS
 
 /**
- * Retrieves the GeoJSON schema from an external URL and returns it as a JSON object
+ * Retrieves the GeoJSON schema from the GeoJSON.json file
  * @return {Promise}
  */
 function getGeoJSONSchema() {
```

Comment:
The `getGeoJSONSchema` method is actually redundant now that the GeoJSON schema is loaded directly from a JSON file. The `require` statement at the top loads the GeoJSON schema and makes it available to the rest of this script, so you don't have to wait for it asynchronously. You can delete this method entirely, and refer directly to the `GeoJSON` constant instead.

This means you can also update this line: https://github.com/digitallinguistics/spec/pull/330/files#diff-a5509d3e2a64ff6e2ca19860925a96a7R55

Reply:
I have removed the method.